core/geometry/src/clipping/ClipPrimitive.ts

```diff
@@ -417,7 +417,8 @@ export class ClipShape extends ClipPrimitive {
   }
   /**
    * Return true if this ClipShape has a local to world transform
-   * @deprecated Use duplicate property [[transformValid]] or type guard [[hasTransformFromClip]] instead.
+   * @deprecated in 5.0 - will not be removed until after 2026-06-13. Use duplicate property [[transformValid]] or
```

Comment:
@dassaf4 I got a lint error for this. Not sure if what I added is accurate. Please confirm.